src/Integration.Vsix/SonarLintDaemon/SonarLintDaemon.cs

```diff
@@ -40,17 +41,20 @@ internal class SonarLintDaemon : ISonarLintDaemon
         private static readonly string DAEMON_HOST = "localhost";
         private static readonly int DEFAULT_DAEMON_PORT = 8050;
 
-        public const string daemonVersion = "4.0.2.2142";
-        private const string uriFormat = "http://repo1.maven.org/maven2/org/sonarsource/sonarlint/core/sonarlint-daemon/{0}/sonarlint-daemon-{0}-windows.zip";
+        public const string DefaultDaemonVersion = "4.0.2.2142";
```

Comment:
naming nitpick: the constants above are UPPER_CASE, the 2 constants below have different casing - should the naming be more uniform?

also: do these need to be public or only internal for tests?

Reply:
The other constants in the project are mixed case. I'll change the ones in lines 41 and 42.

The class is internal so the effective scope is internal. However, I'll change them to internal since they are only exposed for testing.